src/lib/op_stats.h

```diff
@@ -44,13 +45,14 @@ class op_stats : public op {
 public:
 	op_stats(shared_connection c);
 	virtual ~op_stats();
+	virtual int run_client(stats_results& results);
```

Comment:
I agree with Ikehara-san's comment.
That's doing too much only for debugging.
This design is not easy to understand.